pymeasure/instruments/attocube/anc300.py

```diff
@@ -156,26 +174,48 @@ def stop(self):
         self.write('stop')
         self.check_set_errors()
 
+    def move_raw(self, steps):
+        """Move 'steps' steps in the direction given by the sign of the
+        argument. This method assumes the mode of the axis is set to 'stp' and
+        it is non-blocking, i.e. it will return immediately after sending the
+        command.
+
+        :param steps: integer value of steps to be performed. A positive
+            sign corresponds to upwards steps, a negative sign to downwards
```

Comment:
A user of the ANC300 does know, what "upwards" and "downwards" is?

Reply:
I checked the manual and actually the wording used in docstrings matches the Attocube manual. They also write upwards and downwards to specify directions. (This is also the reason the commands are "stepu" and "stepd").

Since I have no rotation stage accessible, and the manual does not mention the rotation sense I would prefer to keep it as is.

Reply:
Ok. If it is in the manual, a user will understand it, so my original question is answered.